src/cascadia/TerminalSettingsEditor/RenderingViewModel.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "RenderingViewModel.g.h"
+#include "ViewModelHelpers.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    struct RenderingViewModel : RenderingViewModelT<RenderingViewModel>, ViewModelHelper<RenderingViewModel>
+    {
+    public:
+        RenderingViewModel(Model::GlobalAppSettings globalSettings);
+        Model::GlobalAppSettings GlobalSettings();

```

Comment:
```suggestion
        Model::GlobalAppSettings GlobalSettings() const;
```
nit: I think you can add a const here?

Reply:
Yep, thanks for the catch! (and the others too)

Reply:
Doesn't actually matter; the Windows Runtime (and COM) doesn't have a notion of const, so really it's never going to care that you've decorated or not-decorated your members